test_util/azure.py

```diff
@@ -35,77 +35,123 @@ def validate_hostname_prefix(prefix):
     assert re.match('^[a-z][a-z0-9-]{1,61}[a-z0-9]$', prefix), 'Invalid DNS prefix: {}'.format(prefix)
 
 
+def check_json_object(obj):
```

Comment:
-ve on this.

If we use type-hints and mypy, then we dont need this. 

And in this code base we already have started using type hints.